lib/parsers/dpx_parser.rb

```diff
@@ -35,18 +35,23 @@ def call(io)
     w = dpx_structure.fetch(:image).fetch(:pixels_per_line)
     h = dpx_structure.fetch(:image).fetch(:lines_per_element)
 
+    display_w = w
```

Comment:
Is there any reason why you moved these lines from their original place? 🤔

Reply:
I noticed that w and h are already being calculated above and the other statements in between do that impact their values. So the display_w and display_h can be initialised as soon as we have the values of h and w ready.